tests/lib/managers/webhooks-test.js

```diff
@@ -0,0 +1,117 @@
+/**
+* @fileoverview Webhooks Manager Tests
+* @author ccheng
+*/
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake,
+	Webhooks,
+	webhooks,
+	WEBHOOKS_ID = '1234',
+	MODULE_WEBHOOKS_PATH = '../../../lib/managers/webhooks';
+
+describe('Webhooks', function() {
+
+	beforeEach(function() {
+		// Setup Environment
+		boxClientFake = leche.fake(BoxClient.prototype);
+		// Register Mocks
+		mockery.enable({
+			useCleanCache: true
+		});
+		mockery.registerAllowable('http-status');
+		mockery.registerAllowable('util');
+		mockery.registerAllowable('../util/url-path');
+		mockery.registerAllowable('../util/errors');
+		// Setup File Under Test
+		mockery.registerAllowable(MODULE_WEBHOOKS_PATH);
+		Webhooks = require(MODULE_WEBHOOKS_PATH);
+		webhooks = new Webhooks(boxClientFake);
+	});
+
+	afterEach(function() {
+		sandbox.verifyAndRestore();
+		mockery.deregisterAll();
+		mockery.disable();
+	});
+
+	describe('createWebhook()', function() {
+
+		it('should make POST call to create webhook', function(done) {
+			var	id = '1234',
+				type = 'file',
+				address = 'https://www.test.com',
+				triggers = ['FILE.DOWNLOADED', 'FILE.PREVIEWED'];
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler').yields();
```

Comment:
This isn't the right behavior for the stub; `defaultResponseHandler` doesn't call the function it's passed, it returns a wrapped version of it.  The stubs should be updated to something like `.returnsArg(0)` for correctness